internal/api/web/views/components/dashboard_components/side_menu.templ

```diff
@@ -0,0 +1,29 @@
+package dashboard_components
+
+templ SideMenu() {
+	<div class="fixed h-full text-center w-48 bg-gray-100">
+		<h1 class="text-3xl mb-4 bg-gray-300 py-2">Quiz Admin</h1>
+		<nav>
+			<ul class="space-y-4">
+				<li>
+					<a href="/dashboard" class="block bg-gray-200 hover:bg-gray-300 py-2">
+						Hjem
+					</a>
+				</li>
+				<li>
+					<a href="/dashboard/leaderboard" class="block bg-gray-200 hover:bg-gray-300 py-2">
+						Toppliste
+					</a>
+				</li>
+				<li>
```

Comment:
Instead of duplicated classes, consider refactoring into a separate component.

Reply:
fe13fda90e2c5325777ebadfcb7bac5a9c374c80